--- client.bal
+++ client.bal
@@ -82,7 +82,7 @@
     resource isolated function post payments/[string id](Payment payload, map<string|string[]> headers = {}) returns PaymentReceipt|error {
         string resourcePath = string `/payments/${getEncodedUri(id)}`;
         http:Request request = new;
-        json jsonBody = jsondata:toJson(payload)
+        json jsonBody = jsondata:toJson(payload);
         request.setPayload(jsonBody, "application/vnd.snowpeak.resort+json");
         return self.clientEp->post(resourcePath, request, headers);
     }