--- client.bal
+++ client.bal
@@ -73,7 +73,6 @@
     resource isolated function delete reservations/[string id](map<string|string[]> headers = {}) returns ReservationReceipt|error {
         string resourcePath = string `/reservations/${getEncodedUri(id)}`;
         return self.clientEp->delete(resourcePath, headers = headers);
     }
-o
     # Snowpeak payment resource
     #
     # + id - Unique identification of reservation